client/app/app.js

```diff
@@ -8,6 +8,9 @@ import Common from "./common/common";
 import PageComponents from "./components/pages";
 import AppMaterial from "./app.material";
 import AppTranslate from "./app.translate";
+import AppXEditableConfig from "./app.xeditable";
+import "angular-xeditable";
```

Comment:
@nicksellen can you hint me how to do this better? I didn't see a way to import its name, seems not to support ES6 style.